### DISCUSSION TITLE
[Feature Request]: Ability to publish stub file

### DISCUSSION THREAD

**Discussion by danie-ramdhani** [rappasoft/laravel-livewire-tables#1478] [Feature Requests]
### Overview

It would be great if we can use our stub to make a small changes whenever the table component is generated

**Comment by lrljoe** [comment]
Just so I have the request clear, are you asking for:

Ability to customise the default stub that's used to generate a table via the make:datatable command?

What're you trying to add as a default out of interest?

**Comment by danie-ramdhani** [comment]
publish the datatable stub console like: https://laravel.com/docs/10.x/artisan#stub-customization

i always using a custom trait in my datatable to load some functions:

```php
class MyTable extends DataTableComponent
{
    use MyTablesConfig;
    ....
```

so, it's better to have published `table.stub` whenever i want to generate a new datatable
